runtime/constants/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub mod time {
 	pub const MILLISECS_PER_BLOCK: Moment = 3000;
 	pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
 	#[allow(clippy::identity_op)]
-	pub const EPOCH_DURATION: BlockNumber = prod_or_fast!(1 * HOURS, 2 * MINUTES);
+	pub const EPOCH_DURATION: BlockNumber = prod_or_fast!(2 * MINUTES, 1 * MINUTES);
```

Comment:
Question: What exactly `EPOCH_DURATION` does ?

Reply:
sets the session time. We use sessions to rotate the validators/authorities.

Reply:
Private/public key pair changes in this interval? Why change the keys ??

Reply:
Session rotation provides a window to remove the offline authorities and add new authorities. No Key pair changes here.